physical/raft/raft_test.go

```diff
@@ -203,6 +205,71 @@ func TestRaft_Backend(t *testing.T) {
 	physical.ExerciseBackend(t, b)
 }
 
+func TestRaft_Backend_LargeValue(t *testing.T) {
+	b, dir := getRaft(t, true, true)
+	defer os.RemoveAll(dir)
+
+	t.Helper()
```

Comment:
This usually goes first right after the signature (i.e. L209).

Reply:
Actually this test itself is not a helper, so it might be better to simply remove `t.Helper()`.